packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -788,6 +790,10 @@ export class FocusZone extends BaseComponent<IFocusZoneProps, {}> implements IFo
   }
 
   private _shouldInputLoseFocus(element: HTMLInputElement, isForward?: boolean) {
+    if (this.props.tabPermission === FocusZoneTabbableElements.inputOnly && !this._processingTabKey) {
```

Comment:
You don't know that the element is an input at this point, this check should be moved down to the if statement below.